mods/weather/init.lua

```diff
@@ -1,11 +1,18 @@
 -- Disable by mapgen or setting
 
 local mg_name = minetest.get_mapgen_setting("mg_name")
-if mg_name == "v6" or mg_name == "singlenode" or
-		minetest.settings:get_bool("enable_weather") == false then
+if minetest.settings:get_bool("enable_weather") == false then
```

Comment:
Needs documentation in `minetest.conf.example` that enable_weather also controls shadows.